Examples/Examples/ContentView.swift

```diff
@@ -103,3 +124,29 @@ private extension WebViewProxy {
 #Preview {
     ContentView()
 }
+
+extension PDFDocument: @retroactive Transferable {
+    public static var transferRepresentation: some TransferRepresentation {
+        DataRepresentation(contentType: .pdf) { pdf in
+                if let data = pdf.dataRepresentation() {
+                    return data
+                } else {
+                    return Data()
+                }
+            } importing: { data in
+                if let pdf = PDFDocument(data: data) {
+                    return pdf
+                } else {
+                    return PDFDocument()
+                }
+            }
```

Comment:
nits: The indentation seems a bit misaligned.
```suggestion
        DataRepresentation(contentType: .pdf) { pdf in
            if let data = pdf.dataRepresentation() {
                return data
            } else {
                return Data()
            }
        } importing: { data in
            if let pdf = PDFDocument(data: data) {
                return pdf
            } else {
                return PDFDocument()
            }
        }
```